reloc.ml

```diff
@@ -433,13 +433,18 @@ let add_reloc_table obj obj_name p =
         | `x64, 0x01 (* IMAGE_REL_AMD64_ADDR64 *) ->
             0x0002 (* absolute, native size (32/64) *)
 
+        | `x86, 0x07 (* IMAGE_REL_I386_DIR32NB *)
+        | `x64, 0x03 (* IMAGE_REL_AMD64_ADDR32NB *) ->
+            0x0007 (* 32nb *)
+
         | `x64, 0x04 (* IMAGE_REL_AMD64_REL32 *)
         | `x86, 0x14 (* IMAGE_REL_I386_REL32 *) when not !no_rel_relocs ->
             0x0001 (* rel32 *)
 
         | `x64, 0x05 when not !no_rel_relocs -> 0x0004 (* rel32_1 *)
         | `x64, 0x08 when not !no_rel_relocs-> 0x0003 (* rel32_4 *)
         | `x64, 0x06 when not !no_rel_relocs-> 0x0005 (* rel32_2 *)
+        | `x64, 0x09 when not !no_rel_relocs-> 0x0006 (* rel32_5 *)
```

Comment:
While we here, might we as well add the `rel32_3` case? I guess that these were included "on demand" in the past, but it does just look to be a set of them!

Reply:
Yep. Will do after I get the existing relocations working properly! (My latest comment has the details).